elk/logstash/Dockerfile

```
@@ -0,0 +1,7 @@
+ARG ELASTIC_VERSION
+
+# https://www.docker.elastic.co/
+FROM docker.elastic.co/logstash/logstash:${ELASTIC_VERSION:-8.17.4}
```

Hadolint (2.12.0)

[error] unexpected ':'
expecting '@', '\', a new line followed by the next instruction, or the image tag

(DL1000)